src/App.js

```diff
@@ -467,6 +476,70 @@ function handleParentWindowMessage(event) {
   }
 }
 
+//send postMessage upon load of script
+//probably best to call this in contacts.js
+//add debugging in to figure out just how things are traveling
+//add finesse side of phonebook
+// finesse api? will have to figure out how user is created d
+function getContactsList(){
+  return contacts_list_global;
+}
+function formatNumbers(contact_list){
+  // starts with ( correct
+  // starts with number, remove first - and add ()
+  // starts with + add () and -
+  var returnlist = [];
+  var numberPattern = /\d+/g;
+  for(var i = 0; i < contact_list.length; i++){
+    var numbers = contact_list[i].number.match(numberPattern);
+    if(numbers == null){
```

Comment:
We need to support both business phone ("phone" field in SNOW) and mobile ("mobile_phone" field in SNOW). Bucher Suter's app does it with little phone icons, a desk phone icon for business phone and mobile phone icon for mobile. Maybe something like that if the text doesn't fit well. Either that or make the text smaller and have one on top of the other, with labels for "business:" and "mobile:"

Reply:
fixed